### PR TITLE
docs: add documentation about the offline docs

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

**Review by bpmct — COMMENTED** [review]
Let's also be sure to include the `CODER_DOCS_URL` docs here. Maybe let's hold off on merging until that is in? Otherwise, LGTM!

**Comment by BrunoQuaresma** [comment]
@bpmct it is merged already. I also updated the docs to include the env variable.

File: docs/install/offline.md [hunk 1]
```diff
@@ -185,3 +185,15 @@ coder:
...
+
+## Offline docs
+
+Coder also provides offline documentation in case you want to host it on your own server. The docs are exported as static files that you can host on any web server, as demonstrated in the example below:
+
+1. Go to the release page. In this case, we want to use the [latest version](https://github.com/coder/coder/releases/latest).
+2. Download the documentation files from the "Assets" section. It is named as `coder_docs_<version>.tgz`.
+3. Extract the file and move its contents to your server folder.
+4. If you are using NodeJS, you can execute the following command: `cd docs && npx http-server .`
+5. Set the `CODER_DOCS_URL` environment variable to use the URL of your hosted docs. This way, the Coder UI will reference the documentation from your specified URL.
```
Comment (bpmct) [inline docs/install/offline.md:197]:
```suggestion
5. Set the [CODER_DOCS_URL](../cli/server#--docs-url) environment variable to use the URL of your hosted docs. This way, the Coder UI will reference the documentation from your specified URL.
```

**Review by bpmct — APPROVED** [review]

File: docs/install/offline.md (hunk 1 above)
Comment (BrunoQuaresma) [reply docs/install/offline.md:197]: I will make a hotfix for this

**Pull request merged**